src/server/models/user.js

```diff
@@ -44,4 +44,12 @@ const UserSchema = new mongoose.Schema({
 const User = mongoose.model("Users", UserSchema);
 
 
+// User.create({
+//     firstname: "Bob",
+//     lastname: "Test",
+//     email: "bobTest@hotmail.com",
+//     password: "122121212",
+//     jobsApplied: [1234131231242]
+// })
```

Comment:
We can re-use sample data as test data for running tests.
You can remove this from the file.